src/test/java/SetCollectionTest.java

```diff
@@ -0,0 +1,46 @@
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetCollectionTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("size()로 Set의 크기 확인")
+    void sizeTest(){
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+
+
+    @ParameterizedTest
+    @ValueSource(ints = {1,2,3})
+    @DisplayName("contains()로 set에 있는 값 확인")
+    void containsTest(int number){
+        assertThat(numbers.contains(number)).isTrue();
```

Comment:
```suggestion
        assertThat(numbers).contains(number);
```
위와 마찬가지로 Assesrtsions에는 다양한 문법을 지원하고 있으니 미션 진행하시면서 필요하실때마다 하나씩 확인해보시면 도움이 되실 것 같아요 😄